modules/web-console/frontend/app-angular/components/form-field/formField.component.scss

```diff
@@ -0,0 +1,176 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+form-field {
+    --required-label-gap: 0.25em;
+    --input-height: 36px;
+    --overlay-item-width: 16px;
+    --overlay-item-gap: 10px;
+
+    &.form-field__icon-error {
+        display: grid;
+        grid-template-areas: 'title title' 'control overlay' 'error-inline error-inline';
+        grid-template-columns: auto min-content;
+
+        .angular-form-field__label {
+            grid-area: title;
+        }
+
+        .angular-form-field__input {
+            grid-column: control / overlay;
+            grid-row: control;
+        }
+    }
+
+    .angular-form-field__input[data-overlay-items-count="1"] {
+        input {
+            padding-right: calc(1 * (var(--overlay-item-width) + var(--overlay-item-gap)) + var(--overlay-item-gap));
+        }
+    }
+
+    .angular-form-field__input[data-overlay-items-count="2"] {
+        input {
+            padding-right: calc(2 * (var(--overlay-item-width) + var(--overlay-item-gap)) + var(--overlay-item-gap));
+        }
+    }
+
+    .input-overlay {
+        grid-area: overlay;
+        display: grid;
+        grid-auto-columns: var(--overlay-item-width);
+        grid-auto-flow: column;
+        padding-right: var(--overlay-item-gap);
+        grid-gap: var(--overlay-item-gap);
+        z-index: 2;
+        // Fixes z-order in Edge
+        transform: translateZ(1px);
+    }
+
+    input, select {
+        box-sizing: border-box;
+        height: var(input-height);
+        padding: 9px 10px;
+        border: solid 1px #c5c5c5;
+        border-radius: 4px;
+        background-color: #ffffff;
+        color: #393939;
+        font-size: 14px;
+        line-height: 16px;
+        width: 100%;
+
+        &[disabled] {
+            opacity: 0.5;
+        }
+
+        &::placeholder {
+            color: rgba(66, 66, 66, 0.5);
+            text-align: left;
+        }
+    }
+    select {
+        -webkit-appearance: unset;
+        appearance: unset;
+        background-image: url(data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHZpZXdCb3g9IjAgMCA4IDQiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0ibTAgMGg4bC00IDR6IiBmaWxsPSIjMzkzOTM5Ii8+PC9zdmc+);
```

Comment:
I would better use normal svg if possible

Reply:
It is "normal svg", just inline. I don't really think it matters in this particular case.